dao/patient/etl-patient-dao.js

```diff
@@ -92,6 +81,142 @@ module.exports = function () {
         return promise;
     }
 
+    function getPatientOncologySummary(request, callback) {
```

Comment:
Move this to patientoncologysummaryservice under /app/services. The reason is, this is legacy code that will be refactored eventually. 
etl-routes will consume this service.